### PR TITLE
[MessageBus]LPS-102995

### PR DESCRIPTION
**Pull request by tinatian**

### CODE REVIEW

**Comment by tinatian** [comment]
ci:test:backend

File: modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/internal/CTMessageBusInterceptor.java [hunk 1]
```diff
@@ -38,19 +35,13 @@
 	public boolean intercept(
 		MessageBus messageBus, String destinationName, Message message) {
 
-		long ctCollectionId = CTCollectionThreadLocal.getCTCollectionId();
-
-		if (ctCollectionId == CTConstants.CT_COLLECTION_ID_PRODUCTION) {
+		if (!destinationName.equals(DestinationNames.SUBSCRIPTION_SENDER)) {
```
Comment (shuyangzhou) [inline modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/internal/CTMessageBusInterceptor.java]: Please pull this out as an OSGi configuration.

**Comment by vicnate5** [comment]
@shuyangzhou Test results look good, there are some unique workflow test failures on Hypersonic ONLY
Probably not related, just highlighting in case
https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/342645/AXIS_VARIABLE=35,label_exp=!master/testReport/junit/com.liferay.portal.workflow.metrics.service.test/WorkflowMetricsSLADefinitionLocalServiceTest/testAddSLADefinition1/

**Comment by shuyangzhou** [comment]
Thanks @vicnate5 !

**Comment by Preston-Crary** [comment]
Please see https://github.com/shuyangzhou/liferay-portal/pull/8610

**Pull request closed without merging**